SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -80,6 +80,9 @@ class CVariable {
   VectorType AuxVar;       /*!< \brief Auxiliar variable for gradient computation. */
   MatrixType Grad_AuxVar;  /*!< \brief Gradient of the auxiliar variable. */
 
+  MatrixType AxiAuxVar;             /*!< \brief Axisymmetric auxiliar variable for gradient computation. */
+  CVectorOfMatrix Grad_AxiAuxVar;   /*!< \brief Gradient of the axisymmetric variables  of the problem. */
+
```

Comment:
I wonder if it would make sense to generalize AuxVar to have any number of variables, instead of adding one more "Aux thing". Do you see any conflict with how they are used?

Reply:
There shouldnt be any conflict...I think this is a good idea.

Reply:
After looking more deeply into the code, other portions of the code use the AuxVar functons. But these are 
unrelated to the axisymmetric terms.  (CElasticity, CAdjEuler).  I think a generalization would be possible.

After there any examples of this type of generalization that I can work with?

Reply:
I don't think there are, but it should be just a matter of replacing the uses of AuxVar by AxiAuxVar, fixing the compilation errors, then deleting the methods dealing with AuxVar, and finally renaming AxiAuxVar by just AuxVar :) or something like that.
If it gets too messy I can help.

Reply:
@pcarruscag Thanks for the review and help!  I have spending some time trying to generalize the functions, but I am running into 2 main issues (I think its only 2).  The main issue moving properly handing the change from vectors to matrix of vectors.   If you had time, I would very much appreciate your help :)   

I haven't pushed the current code, since its hacked together at the moment.